thunder/extend/__init__.py

```diff
@@ -458,6 +458,8 @@ def resolve_executors(executors: None | Sequence[Executor | str]) -> tuple[Execu
                 continue
             else:
                 resolved_executors.append(ex)
+        elif e is None:
```

Comment:
No. If we want to do this, we need to check for "other than executor" and error with what we got.
`executors=(my_exec, thunder.get_default_executors())`

Reply:
What do you mean by "other than executor"? As in, check if it isn't a string or an executor, not just None?

Reply:
Fixed. I think the `None` case is going to be really common though, so I retained it for a more informative error message.

Reply:
I would prefer the None to show the same error message, because it expresses that it should be an executor (with the string being the lazy option) in that list and it saves unneeded special casing.